public/app/features/auth-config/state/actions.ts

```diff
@@ -31,13 +31,13 @@ export function loadSettings(): ThunkResult<Promise<Settings>> {
   };
 }
 
-export function loadProviders(): ThunkResult<Promise<SSOProvider[]>> {
+export function loadProviders(provider = ''): ThunkResult<Promise<SSOProvider[]>> {
   return async (dispatch) => {
     if (!config.featureToggles.ssoSettingsApi) {
       return [];
     }
-    const result = await getBackendSrv().get('/api/v1/sso-settings');
-    dispatch(providersLoaded(result));
+    const result = await getBackendSrv().get(`/api/v1/sso-settings/${provider}`);
```

Comment:
If `provider` is empty and we want the list of all providers then the URL should not contain the ending `/`. So for requesting the list of providers this should be used `/api/v1/sso-settings` instead of `/api/v1/sso-settings/`. If we include the `/` at the end then the backend will interpret it as a single provider request with an empty provider name (and it will return an error).

Reply:
Good catch, I'll fix it.